catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrFilterDelegate.java

```diff
@@ -765,6 +773,33 @@ public SolrQuery contains(String propertyName, String wkt) {
     return operationToQuery("Contains", propertyName, wkt);
   }
 
+  public static Geometry removeHoles(final Geometry geo) {
```

Comment:
So, obviously you saw some failures with this and had to remove them, but I am having trouble following why JTS doesn't handle these polygons.  What was the error you were seeing?  One of the assertions failing or an exception being thrown by JTS?

Reply:
I tried to explain the issue in the second bullet point of the PR description. You've reminded me that I still need to add some comments to the code, too. This fix is to prevent this exception from being thrown: https://github.com/locationtech/spatial4j/blob/2926812ae302c6e0f24fb5995b1fd24be7fe0b56/src/main/java/org/locationtech/spatial4j/shape/jts/JtsGeometry.java#L492

Reply:
https://github.com/locationtech/spatial4j/issues/220
Trying to get some more information about the issue here.